### PR TITLE
AVO2-628: Added FlowPlayer component to EditCollection page & added "Knippen" modal.

### PR DESCRIPTION
**Pull request by benjaminnaesen**

### CODE REVIEW

File: src/collection/components/CollectionFragment.tsx [hunk 1]
```diff
@@ -29,6 +32,7 @@ import ControlledDropdown from '../../shared/components/ControlledDropdown/Contr
 import { DataQueryComponent } from '../../shared/components/DataComponent/DataQueryComponent';
 import toastService, { TOAST_TYPE } from '../../shared/services/toast-service';
 import { GET_ITEM_META_BY_EXTERNAL_ID } from '../graphql';
+import { CutFragmentModal } from './';
```
Comment (andrych) [inline src/collection/components/CollectionFragment.tsx]: `import CutFragmentModal from './CutFragmentModal'`

**Comment by bertyhell** [comment]
image

**Comment by bertyhell** [comment]
mss nog eens onderzoeken waarom dit item op de edit pagina de NotFound component toont
http://localhost:8080/collectie/1131711/bewerk
image

**Comment by bertyhell** [comment]
thumbnail play button should have bg circle to make sure it is visible on all backgrounds
image

image

**Comment by bertyhell** [comment]
Make the whole thumbnail clickable to init the flowplayer 
image

**Pull request merged**